core/service/Cargo.toml

```diff
@@ -32,6 +32,7 @@ transaction_pool = { package = "substrate-transaction-pool", path = "../../core/
 rpc = { package = "substrate-rpc-servers", path = "../../core/rpc-servers" }
 tel = { package = "substrate-telemetry", path = "../../core/telemetry" }
 offchain = { package = "substrate-offchain", path = "../../core/offchain" }
+parity-multiaddr = { package = "parity-multiaddr", version = "0.5.0" }
```

Comment:
I don't think you're using this dependency, and if you are, `multiaddr` is reexported by `network`.